### PR TITLE
ci: add unit testing on navigation

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- CI

#### What is expected?

Add tests for Navigation.

### CODE REVIEW

File: packages/plus/src/components/Navigation/__tests__/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  //     name: 'Toggle navigation expand/collapse',
+  //   })
+  //   expandButton.click()
+
+  //   expect(asFragment()).toMatchSnapshot()
+  //   const item = screen.getAllByText('item1')[0]
+
+  //   item.click()
+  //   expect(asFragment()).toMatchSnapshot()
+  // })
```
Comment (radhi-nasser-scaleway) [inline packages/plus/src/components/Navigation/__tests__/index.test.tsx:86]: to remove

**Review by radhi-nasser-scaleway — APPROVED** [review]

**Review by JulienSaguez — APPROVED** [review]

File: packages/plus/src/components/Navigation/NavigationContent.tsx [hunk 2]
```diff
@@ -316,6 +316,9 @@ export const NavigationContent = ({
                   sentiment="neutral"
                   size="small"
                   icon={expanded ? 'arrow-left-double' : 'arrow-right-double'}
+                  aria-label={
+                    expanded ? 'Collapse navigation' : 'Expand navigation'
```
Comment (alexandre-combemorel) [inline packages/plus/src/components/Navigation/NavigationContent.tsx:320]: no variable to pass translation ? agree its probably over loaded, just wondering what is the practice so far?

File: packages/plus/src/components/Navigation/__tests__/index.test.tsx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    expect(asFragment()).toMatchSnapshot()
+
+    const collapseButton = screen.getByRole('button', {
+      name: 'Collapse navigation',
+    })
+    collapseButton.click()
+    await waitFor(() => {
+      expect(
+        screen.getByRole('button', { name: 'Expand navigation' }),
+      ).toBeInTheDocument()
```
Comment (alexandre-combemorel) [inline packages/plus/src/components/Navigation/__tests__/index.test.tsx:58]: better to use toBeVisible, if the button is hidden is css it still works

**Review by alexandre-combemorel — APPROVED** [review]

File: packages/plus/src/components/Navigation/NavigationContent.tsx (hunk 2 above)
Comment (matthprost) [reply packages/plus/src/components/Navigation/NavigationContent.tsx:321]: true I will reuse the one from tooltip

**Pull request merged**